moda-contracts/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "moda-contracts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
I think we should leave docs in the readme to show how to call this from the root directory

Reply:
Done,  let me know if that's alright, i'm not exactly sure how descriptive it needs to be.